src/query/storages/common/table_meta/src/meta/v2/statistics.rs

```diff
@@ -69,20 +89,20 @@ impl ColumnStatistics {
         distinct_of_values: Option<u64>,
     ) -> Self {
         Self {
```

Comment:
Maybe we could add debug_assert to assume that:

min, max are both same enum types or at least one of them is null.